lib/game_teams_manager.rb

```diff
@@ -18,6 +19,12 @@ def create_game_teams(game_teams_table)
     end
   end
 
+  def total_score(filtered_game_teams = @game_teams)
+    filtered_game_teams.count do |game_team|
+      game_team.goals
+    end
+  end
+
```

Comment:
This is a method that exists in games manager. I'm wondering if it's more efficient code-wise to just call on that method? Maybe just make a note of it for now.

Reply:
Well, I don't see it in the games manager... I see lowest_total_score and highest_total_score